src/Entity/SmsMessage.php

```diff
@@ -444,7 +448,11 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
     $fields['recipient_entity'] = BaseFieldDefinition::create('dynamic_entity_reference')
       ->setLabel(t('Recipient entity'))
       ->setDescription(t('The entity who received the SMS message.'))
-      ->setRequired(FALSE);
+      ->setRequired(FALSE)
+      ->setSettings([
+        'exclude_entity_types' => TRUE,
```

Comment:
Its weird, I cannot reproduce the fatal after re-downloading DER. Seems like co-incidence.

Looks like the settings are no longer required, so I have removed them from this PR. This PR now focuses on fixing the tests.

Posted an apology to DER https://www.drupal.org/node/2829049#comment-11793284